crates/matrix-sdk/src/event_cache/room/mod.rs

```diff
@@ -1484,10 +1549,9 @@ mod tests {
                         new: ChunkIdentifier::new(0),
                         next: None,
                     },
-                    Update::NewItemsChunk {
-                        previous: Some(ChunkIdentifier::new(0)),
-                        new: ChunkIdentifier::new(1),
-                        next: Some(ChunkIdentifier::new(0)),
+                    Update::PushItems {
+                        at: Position::new(ChunkIdentifier::new(0), 0),
+                        items: iter::repeat_n(event, DEFAULT_CHUNK_CAPACITY + 1).collect(),
```

Comment:
Hmm, the test shouldn't require any change, since this was testing for cycles. If it failed somehow before this change, it means the incremental rebuilding doesn't correctly handle cycles. Can you revert this please? :nerd_face: